ConfigurationSystem/Client/Utilities.py

```diff
@@ -273,6 +273,13 @@ def addToChangeSet( entry, changeSet ):
           newMaxCPUTime = queueInfo.get( 'GlueCEPolicyMaxCPUTime', '' )
           if newMaxCPUTime == "4" * len( newMaxCPUTime ) or newMaxCPUTime == "9" * len( newMaxCPUTime ):
             newMaxCPUTime = ''
```

Comment:
You know there was a controversy with respect to 'default' values in the Glue Schema at some point. Those lead to interesting discrepancy between the OSG and LCG... Basically, some think that putting 99999 or 44444 as default is reasonable. Those mean there are no known values. It's awful and breaks the OSG brokers...